module-platform/linux/tests/genlfstestimg.sh

```diff
@@ -27,19 +26,15 @@ if [ ! -d "${SYSROOT}/sys" ]; then
 	exit -1
 fi
 
-_REQ_CMDS="sfdisk truncate"
-for cmd in $_REQ_CMDS; do
-	if [ ! $(command -v $cmd) ]; then
-		echo "Error! $cmd is not installed, please use 'sudo apt install' for install required tool"
-		exit -1
-	fi
-done
-truncate -s $IMAGE_SIZE $IMAGE_FILE
+. $(dirname $0)/../../../tools/locate_bins.sh
```

Comment:
this is error-prone when the directory changes. Use:
```
L_GIT_DIR=$(git rev-parse --show-toplevel)
```
(from:  config/style_check_hook.sh )

Reply:
OK.